salt/modules/boto_cloudfront.py

```diff
@@ -456,3 +460,1230 @@ def update_distribution(
         )
 
     return {'result': True}
+
+
+def get_distribution_v2(region=None, key=None, keyid=None, profile=None, **kwargs):
```

Comment:
What sort of items should a person be passing here via kwargs? Can they be documented? If not, can this be removed entirely?